lib/scripts/controllers/options.js

```diff
@@ -161,6 +170,23 @@ export class OptionCtrl {
     return new Array(Math.ceil((Object.keys(hash).length)/4));
   }
 
+  randomlySelectTenWords(translatedWords){
+    var keys = Object.keys(translatedWords);
+    this.translatedWordsExist = keys.length === 0 ? false : true;
+    var result = {};
+    for (var i=0; i<10; i++) {
+      let randomKey = keys[Math.round(Math.random() * (keys.length - 0) + 0)];
```

Comment:
If keys.length is small, you may have a high probability of selecting the same word multiple times.